### ISSUE TITLE
PlayerTransitionClose Lines (and 3 other suggestions)

### ISSUE THREAD

**Issue by MisterMayoNL** [Tsuey/L4D2-Community-Update#431]
### Suggestion

Some of the PlayerTransitionClose lines are not fitting for survivors. Some of the lines present are meant for when everyone survives instead of with a casualty. Below are the lines I don't think are fitting and I propose should be added to the regular PlayerTransition instead.

Coach:
	TransitionClose08  //Touchdown!
	TransitionClose10  //That was some epic shit. Right there.

Nick:
	TransitionClose04  //That my friends, is how you do it!
	TransitionClose05  //Happy New Year!

Ellis:
	TransitionClose01  //YEAH! _(I find this line really weird tbh. Sounds more like a audio test to me. I'd personally say remove this one entirely)_
	TransitionClose07  //Elvis has left the building!

Rochelle: _(also has unused line TransitionClose03 which could be added to the regular PlayerTransition)_
	TransitionClose06  //Live and in the flesh!
	TransitionClose08  //We made it uh huh. We made it oh yeah (sung)
	TransitionClose07  //All bark, no bite!

Francis:
	PlayerTransitionClose02  //Heh heh, well all right!

Some other misc suggestions I have:

- Add FriendlyFireC102 to PlayerFriendlyFireC1Gambler (I think someone just heard this one too often pre-TLS and decided to get rid of it once and for all)

- Add HealedByFriend for the L4D2 cast when healing eachother

- Add missing lines to game_sounds_producer (ChargerRunningWithPlayer01 & FriendlyFireBile lines). Ellis has these lines in his game_sounds_mechanic (only one of his friendlyfirebile lines is used, but all 5 are in the file), so I don't see why Rochelle shouldn't have these ones as well. Even if they're not used in the vanilla game, they can still be used in mods

**Comment by CubaOfficial** [comment]
There really isn't anything that proves what lines are meant for what context since they are all labeled the same. I'd personally go off the day 1 talker to see what was "intended" by Valve.

**Comment by MisterMayoNL** [comment]
Yes, there's nothing that "technically" proves this, but I think everyone agrees that Nick saying "That my friends, is how you do it!" when there are 2 casualties is not appropriate for the situation.

**Comment by MisterMayoNL** [comment]
Also, if you go by your logic, I don't think Valve ever "intended" to have the CSS weapons available for everyone. Or with reimplementing unused lines, who knows if Valve "intended" to not use these for a reason. I don't think this is an argument that holds much water

**Comment by CubaOfficial** [comment]
That's not what I was trying to imply, I was only referring to the talker. Don't take my feedback in bad faith.

**Comment by MisterMayoNL** [comment]
Sorry if sounded more hostile. Not my intention. And yes I do have this fixed in my own talker. I just think these changes have enough merit to them to apply these in the base game.

For Coach (and to some extent Rochelle) you could argue that they tried to downplay the situation. Nick bein selfish is shown in his transition line if he's the only one left "At least I made it", but the other one seems really odd for a line if he's supposed to be selfish. He would rather just say that he made it, instead of weirdly complimenting the others when ut went bad

**Comment by xanaguy** [comment]
- The lines for PlayerTransitions will not be removed. 

 "Add FriendlyFireC102 to PlayerFriendlyFireC1Gambler (I think someone just heard this one too often pre-TLS and decided to get rid of it once and for all)"

- Not sure why that was missing, but I've added this to the PlayerFriendlyFireC1Gambler response. Thanks for letting me know.

Add HealedByFriend for the L4D2 cast when healing eachother"

- This will result in the current healing lines being interrupted, if added on its own, and plainly. Further investigation will need to be conducted to see if there's any compatibility, even with other healing related lines.

"Remove Hurrah13 from PlayerHurrahProducer (mentions closing the saferoom door, which doesn't make sense mid-game)"

As this is already included in the general PlayerTransition response, I have removed Hurrah13 from the Hurrah response

"Remove spliced lines in L4D1PlayerFriendlyFireFromFrancisNamVet. The three FriendlyFireFrancis lines are just spliced lines of existing friendlyfire lines with Bill saying Francis editing into them. It is beyond me why these are used, as we have non-spliced FF lines of these (NamVet_FriendlyFireFrancis)"

- Things seem fine as they already are. No action necessary... yet.

"Add missing lines to game_sounds_producer (ChargerRunningWithPlayer01 & FriendlyFireBile lines). Ellis has these lines in his game_sounds_mechanic (only one of his friendlyfirebile lines is used, but all 5 are in the file), so I don't see why Rochelle shouldn't have these ones as well. Even if they're not used in the vanilla game, they can still be used in mods. ChargerRunningWithPlayer01 and FriendlyFireBile03 could be used for Teamkill lines for example."

- Sound scripts are a separate type of issue and will need to be looked into further.
